src/raster/r.timeofconcentration/r.timeofconcentration.html

```diff
@@ -99,8 +99,12 @@ <h2>REFERENCES</h2>
 </ol>
 
 <h2>SEE ALSO</h2>
-<p><a href="https://grass.osgeo.org/grass-stable/manuals/r.watershed.html">r.watershed</a>,<br>
-<a href="https://grass.osgeo.org/grass64/manuals/addons/r.stream.distance.html">r.stream.distance</a></p>
+<em>
+<a href="https://grass.osgeo.org/grass-stable/manuals/r.watershed.html">r.watershed</a>,
+<a href="https://grass.osgeo.org/grass64/manuals/addons/r.stream.distance.html">r.stream.distance</a>,
+<a href="https://grass.osgeo.org/grass64/manuals/addons/r.curvenumber.html">r.curvenumber</a>,
+<a href="https://grass.osgeo.org/grass64/manuals/addons/r.runoff.html">r.runoff</a>
```

Comment:
Is grass64 the latest available version for that page?

Reply:
In fact, the url with grass64 doesn’t exist, grass-stable does for at least one of them


```suggestion
<a href="https://grass.osgeo.org/grass-stable/manuals/addons/r.stream.distance.html">r.stream.distance</a>,
<a href="https://grass.osgeo.org/grass-stable/manuals/addons/r.curvenumber.html">r.curvenumber</a>,
<a href="https://grass.osgeo.org/grass-stable/manuals/addons/r.runoff.html">r.runoff</a>
```